graalvm-native-image-app/build.gradle

```diff
@@ -89,7 +89,7 @@ dependencies {
     implementation project(':driver-reactive-streams').sourceSets.test.output
     implementation "org.mongodb:mongodb-crypt:$mongoCryptVersion"
     implementation 'org.slf4j:slf4j-api:2.0.12'
-    implementation 'ch.qos.logback:logback-classic:1.5.3'
+    implementation 'ch.qos.logback:logback-classic:1.5.6'
```

Comment:
`$logbackVersion` does not resolve here, so continuing to specify directly in this spot.